resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -0,0 +1,224 @@
+metadata:
+  name: cloudFoundryDeploy
+  description: Deploys an application to cloud foundry
+  longDescription: |
+    Deploys an application to a test or production space within Cloud Foundry.
+spec:
+  inputs:
+    params:
+      - name: apiEndpoint
```

Comment:
I think we have to adopt the naming to be backwards compatible. Before it was cfApiEndpoint or cloudFoundry/apiEndpoint which could become an alias.

Reply:
same below

Reply:
done, we have now the same aliasing like described [here](https://github.com/SAP/jenkins-library/blob/master/vars/cloudFoundryDeploy.groovy#L162), except for the credentialId.